### DISCUSSION TITLE
Charge VAT at point of sale

### DISCUSSION THREAD

**Discussion by mattgperry** [community/community#11006] [Other Feature Feedback, Questions, & Ideas]
I was under the impression that as I was offering sponsorship that I could declare this merely as income tax, but actually offering any kind of "benefit" tier qualifies this income as VATable.

Showing us where sponsors come from is a good first start but is the bare minimum we need to legally offer benefits. The overhead involved in making sure all this VAT gets where it should go to is far too high, plus sales tax should be charged as per the rate in the sponsor's country rather than from our own pocket.

I'm currently in the process of moving off Github to make sure I can continue to provide benefits legally. Github collecting VAT on top of the base sponsor rate would be a much-needed improvement. Indeed before the new announcement of providing origin information I'm not sure how anyone provided benefits legally on Github Sponsors. Whereas now it's possible to do it legally but not with acceptable financial and admin overhead.

**Comment by glyph** [comment]
I'm in the process of setting up my sponsorship profile and I was very concerned to see this thread https://twitter.com/rafaelcodes/status/1365795326418948097 about this topic, after hearing from a friend in Germany that they can't offer rewards and will need to shut down their profile if they end up collecting more than relatively small threshold of income due to German tax law. I thought that was unfortunate for Europeans, but perhaps Github was thoughtlessly optimized for Americans, which I am.

But then, per Github's own documentation: https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/tax-information-for-github-sponsors 

> For example, if you are a maintainer in the United States and you provide a taxable benefit to a Sponsor in Germany, German sales tax would apply

Browsing around the API it doesn't seem like any KYC information is passed through, making it impossible to *tell* if my sponsors are in Germany, or indeed if they're in Iran or Russia. Given that Github is a US-based entity I am assuming you're handling things like OFAC compliance, but the idea that Github is acting as a "payment processor" here but not passing through the information that payment processors typically do seems like it would create a bottomless pit of potential regulatory hassles.
